[RUMF-1262] add centralized documentation on page activity (#14097)
DataDog/documentation: content/en/real_user_monitoring/browser/tracking_user_actions.md

## What interactions are being tracked?

The Browser SDK automatically tracks clicks. A click action is created if **all** of the following conditions are met:

> * Activity following the click is detected. See [How is page activity calculated?][4] for details.

Reviewer: ```suggestion
* Activity following the click is detected. See [How page activity is calculated][4] for details.
```

### How action loading time is calculated

> The RUM Browser SDK watches the page activity following every click. The action is considered finished when the page has no more activity. See [How page activity is calculated][4] for details.

Reviewer 2: ```suggestion
The Browser RUM SDK watches the page activity following every click. The action is considered finished when the page has no more activity. See [How page activity is calculated][4] for details.
```

> The RUM Browser SDK is watching the page activity following every clicks. The action is considered finished once the page has no more activity. See [How is page activity calculated?][4] for details.

Reviewer: ```suggestion
The RUM Browser SDK watches the page activity following every click. The action is considered finished when the page has no more activity. See [How page activity is calculated][4] for details.
```

[1]: /real_user_monitoring/browser/modifying_data_and_context/
[2]: /real_user_monitoring/browser/data_collected/#default-attributes
[3]: https://github.com/DataDog/browser-sdk/blob/main/CHANGELOG.md#v2160

> [4]: /real_user_monitoring/browser/monitoring_page_performance/#how-is-page-activity-calculated

Reviewer: ```suggestion
[4]: /real_user_monitoring/browser/monitoring_page_performance/#how-page-activity-is-calculated
```
